src/In.ProjectEKA.HipService/DataFlow/DataFlowClient.cs

```diff
@@ -33,19 +33,18 @@ public class DataFlowClient
             IEnumerable<Entry> data,
             KeyMaterial keyMaterial)
         {
-            var url = await centralRegistryClient.GetUrlFor(dataRequest.ConsentManagerId);
-            url.MatchSome(async providerUrl => await PostTo(providerUrl,
-                dataRequest.ConsentId,
+             await PostTo(dataRequest.ConsentId,
```

Comment:
Codacy found an issue: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=5745784)